### PR TITLE
Updating DDP Docs

### PR DESCRIPTION
**Pull request by Eoghan1232**
version 1.0.0.0 of DDP tool doesn't support E810 Series NICs
Updating Docs to make note of this.

@zshi-redhat

### CODE REVIEW

**Comment by Eoghan1232** [comment]
Bit of background on this, Tried to use DDP profiles whilst using an older image, which had ddptool version 1.0.0.0. It did not working when using a E810 Series NIC.

Does the doc need to be updated to make note of this? 

The image update in this PR fixed the issue.
https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/348

File: docs/ddp/README.md
Comment (martinkennelly) [inline docs/ddp/README.md:32]: I think, it would be better to specify the min version of this software that will work for E810 instead of naming DDPTool which is an internal component and from a user perspective is transparent.

Comment (Eoghan1232) [reply docs/ddp/README.md:32]: Do you mean the min version of SRIOV DP?
if so, it would be the version before #348  was merged.
As it was the old images that contained an old dpptool version.

Comment (martinkennelly) [reply docs/ddp/README.md:32]: yes, min version of SRIOV DP. The latest tagged release 3.3.2 should be the min version which support E800 series DDP.
>Also, it was the ddptool that was the issue, and why it didn't work. so specifying the version of DP I think is irrelevant.

DDPTool is included in SRIOV DP (see images folder). From a user perspective, they don't care about version of DDPTool  version. We do not want users to mix and match this tool but take the DDPTool version which comes with this software, so from their perspective, they only care about what ver of DP works with E800 series.

Comment (Eoghan1232) [reply docs/ddp/README.md:32]: Okay I understand now, that makes sense.
Since the "fix" was done recently and is not in the 3.3.2 release. Should this doc update be put on hold till next release?

Comment (Eoghan1232) [reply docs/ddp/README.md]: Done.

**Review by martinkennelly — APPROVED** [review]

**Comment by adrianchiris** [comment]
Mellanox CI issues not related to this PR. moving forward with merge

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**
